db/migrate/20220304145225_create_places.rb

```diff
@@ -0,0 +1,33 @@
+class CreatePlaces < ActiveRecord::Migration[6.1]
+  def change
+    create_table :places do |t|
+      t.string :name, limit: 100, null: false
+      t.integer :type
```

Comment:
Also we have AccommodationType model, maybe change to `type_id`, too @vova??

Reply:
u r right!

Reply:
Update with `accomodation_type_id`
Addressed her https://github.com/vova/war_shelter/pull/22/commits/35ffc92b4826db920f03eeb0ea3f500850cd430a